README.md

```diff
@@ -29,10 +29,19 @@ Please see the following for detailed descriptions of each type:
    ```
    ./generate-go.sh TARGET_PATH
    ```
+   
+### Java
+
+1. Build the go code first (see above) so that all the imports are available
```

Comment:
Done (but if that was the "standard" then I have to say it was non-trivial to implement with vi or emacs).